workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -195,8 +197,34 @@ public bool TryGetEntity(long entityId, out Entity entity)
             return entityMapping.TryGetValue(entityId, out entity);
         }
 
-        public void AddGameObjectEntity(Entity entity, GameObject gameObject)
+        public bool TryGetEntityId(Entity entity, out long entityId)
         {
+            if (!entityManager.HasComponent<SpatialEntityId>(entity))
+            {
+                entityId = 0;
+                return false;
+            }
+
+            entityId = entityManager.GetComponentData<SpatialEntityId>(entity).EntityId;
+            return true;
+        }
+
+        public void AddGameObjectToEntity(Entity entity, string prefabName, Vector3 position, Quaternion rotation, ViewCommandBuffer viewCommandBuffer)
```

Comment:
Should this return the `gameObject` reference?

Reply:
Obsolete now as method has been removed.